### PR TITLE
Fusion doc updates

### PR DESCRIPTION
**Pull request by Stafeeva**
- SubApps
- Workflows

### CODE REVIEW

**Review by adulbrich — COMMENTED** [review]
put the canvas section above the step section 👍

**Review by adulbrich — DISMISSED** [review]

File: docs/src/main/paradox/docs/fusion/architecture.md [hunk 1]
```diff
@@ -28,7 +28,17 @@ Expect the SubApp feature and its functionality will expand and change in the ne
...
+
+## Customization
+
+You can customize the Header of Nexus Fusion by setting up the following environment variables:
+
+- `LOGO_IMG`: Url for an image to be used as application logo in the Header, for example, `https://drive.boogle.com/jnsjdnsjs`
+- `LOGO_LINK`: Url for the logo, for example, `https://www.epfl.ch`
+- If you use Nexus Forge, it is possible to include a `Forge templates` button by providing the url as `FORGE_LINK`, for example, `https://some-url.hi`
+
+The full list of environment variables can be found @link:[here](https://github.com/BlueBrain/nexus-web/blob/main/README.md#env-variables-list).
```
Comment (umbreak) [inline docs/src/main/paradox/docs/fusion/architecture.md:44]: I think the link should be any of the following:
- https://github.com/BlueBrain/nexus-web#env-variables-list
- https://github.com/BlueBrain/nexus-web/blob/main/README.md#env-variables-list

**Review by umbreak — CHANGES_REQUESTED** [review]

File: docs/src/main/paradox/docs/fusion/architecture.md (hunk 1 above)
Comment (Stafeeva) [reply docs/src/main/paradox/docs/fusion/architecture.md:44]: Looks like we are using the second option here already?

Comment (umbreak) [reply docs/src/main/paradox/docs/fusion/architecture.md]: That doesn't seem to work for some weird reason. Please remove the ```@link``` just for that one

**Pull request merged**